### PR TITLE
feature/A2ZO-25:Add React Project

### PR DESCRIPTION
**Pull request by AJakif**

### CODE REVIEW

File: client-ui/src/app/app.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function app() {
```
Comment (deba1) [inline client-ui/src/app/app.tsx]: Always keep the name of Custom component pascal case.
```suggestion
function App() {
```

File: client-ui/src/assets/i.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```
Comment (deba1) [inline client-ui/src/assets/i.tsx]: To keep the folder structure, you can create `.gitkeep` file in that folder. No need to make temporary file.

**Review by deba1 — CHANGES_REQUESTED** [review]
Good work

**Review by MahdiFaheem — CHANGES_REQUESTED** [review]
Remove the following files `favicon.ico`, `logo192.png` and `logo512.png`.

**Review by kanismim — APPROVED** [review]

**Review by MahdiFaheem — APPROVED** [review]

**Review by deba1 — APPROVED** [review]

**Pull request merged**